workflows/proteomics/metaquantome-function/README.md

```diff
@@ -0,0 +1,4 @@
+# metaQuantome 2: Function
+metaQuantome software suite Easterly et al. 2019 was developed by the Team for quantitative and statistical analysis of metaproteomics data. For taxonomic and functional expression analysis within the microbial community, metaQuantome leverages peptide-level quantitative information to generate visual outputs for data interpretation. It also generates outputs that help in understanding the taxonomic contribution to a selected function as well as functions expressed by selected taxonomic group.
```

Comment:
This paragraph doesn't explain what the WF is doing or how that's related to the software suite.